pom.xml

```diff
@@ -7,11 +7,11 @@
     <artifactId>keycloak-metrics-spi</artifactId>
     <groupId>org.jboss.aerogear</groupId>
     <packaging>jar</packaging>
-    <version>3.0.0-SNAPSHOT</version>
+    <version>3.0.1-SNAPSHOT</version>
```

Comment:
think we should bump to a `4.x` instead just a bugfix, because with the switch to keycloak 22 wont be compatible with keycloak < 22

Reply:
That's just what IntelliJ made of it. Hadn't noticed it.
Replaced with single class imports.

Reply:
Agree.
Bumped to 4.0.0.